### ISSUE TITLE
Unable to connect, retrys connection anyways saying "buildVersion out of date."

### ISSUE THREAD

**Issue by notcat** [thomas-crane/nrelay#59]
### Version info
 + nrelay version: `4.3.0`
 + RotMG version: `X20.1.0`

### Steps to reproduce
1. Start nrelay

### Result
```
The bot errors with the error:
[18:59:29 | Main Client]      Connected to USWest!
[18:59:30 | Main Client]      Received socket error: read ECONNRESET
[18:59:30 | Main Client]      The connection to USWest was closed.
[18:59:30 | Main Client]      Connecting in 0.8 seconds.
```
And doesn't stop repeating. I have a brand new, vanilla copy of this, followed the directions on the README.md exactly, but receive this error. Thank you.

**Comment by Maxi35** [comment]
First of all please consider updating to nrelay version v8 it is much better. Secondly deca removed long ago the ability to update the build version when getting the error so you have to update it manually, and right now the version is "1.0.0.0"

**Comment by steele123** [comment]
Yeah, only way to get an accurate buildversion ATM is downloading the swf and looking at the new "Client Version" deca added in the parameters. So a few things about nrelay would need to be updated to fix this.

**Comment by VadimAlexeev0** [comment]
Is there any update to how to fix this issue? I tried to manually update from the swf but the build version keeps reverting to "server.update_client"
